packages/dev/docs/pages/releases/2024-07-20.mdx

```diff
@@ -0,0 +1,127 @@
+  - Cap overlay max height to `100vh` on initial render - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6525)
+  - Fix virtualizer persisted keys with drag and drop - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6644)
+  - Fix divide by zero in GridLayout - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6676)
+  - Fix incremental layout with variable heights - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6588)
+- Misc
+  - Ensure Section has the `[data-rac]` data attribute - [@amitdahan](https://github.com/amitdahan) - [PR](https://github.com/adobe/react-spectrum/pull/6452)
+  - Fix Tailwind plugin options type - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6515)
+  - Fix Compat for React 19 types - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/6632)
+  - Add `nonce` parameter in `LocalizedStringProvider` - [@Julienng](https://github.com/Julienng) - [PR](https://github.com/adobe/react-spectrum/pull/6219)
+  - Fix synthetic links not using `useHref` - [@joshuajaco](https://github.com/joshuajaco) - [PR](https://github.com/adobe/react-spectrum/pull/6346)
+  - Fix source maps in  `@react-aria` - [@oedotme](https://github.com/oedotme) - [PR](https://github.com/adobe/react-spectrum/pull/6368)
+  - Cache locale dictionary to prevent unnecessary re-renders - [@IgnusG](https://github.com/IgnusG) - [PR](https://github.com/adobe/react-spectrum/pull/6185)
+
+## Docs
+- Update `useToast` docs - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/6468)
+- Add Breadcrumb render props to docs - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6489)
+- Add Ripple button example to docs - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/6579)
+- Add example customizing `selectvalue` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/6425)
+- Fix various doc examples for accessiblity - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6607)
+- Fix typo in interactions docs - [@jvliwanag](https://github.com/jvliwanag) - [PR](https://github.com/adobe/react-spectrum/pull/6698)
+- Fix typo in DropZone docs - [@wojtekmaj](https://github.com/wojtekmaj) - [PR](https://github.com/adobe/react-spectrum/pull/6334)
+
+## Under Construction
+
+- Toast (Beta)
+  - Improve Toast focus management - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/6223)
+
+- Color (RC)
```

Comment:
Are the color components going to RC or GA?

Reply:
Decided during planning that we're moving it to RC so I'll leave it under the "Under Construction" category